2018-10-25-Ticha-Review

```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.  
```

Comment:
Please format this article using Markdown. This includes using simple codes to italicize or bold titles, adding links, and adding a YAML header. The instructions are here:

http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/